kuma/users/models.py

```diff
@@ -152,6 +152,10 @@ class User(AbstractUser):
         blank=True,
         validators=[WEBSITE_VALIDATORS['github']],
     )
+    is_github_url_public = models.BooleanField(
+        _(u'Public Github URL'),
+        default=False,
```

Comment:
Is ``False`` the right default value? ``True`` matches current behavior, ``False`` changes to an opt-in. I can see either way. @stephaniehobson ?

Reply:
I believe @stephaniehobson has verbally confirmed that ``default=False`` is the right value, and requiring existing users to opt-in again is OK.